js/packages/mobile-wallet-adapter-protocol/src/types.ts

```diff
@@ -80,12 +99,30 @@ export interface GetCapabilitiesAPI {
 export interface ReauthorizeAPI {
     reauthorize(params: { auth_token: AuthToken; identity: AppIdentity }): Promise<AuthorizationResult>;
```

Comment:
Do we want a deprecation warning on `reauthorize`? I noticed that clientlib-ktx [doesn't have one.](https://github.com/solana-mobile/mobile-wallet-adapter/blob/main/android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/LocalAdapterOperations.kt#L48C1-L49C1)